test/test_tk205.py

```diff
@@ -89,6 +92,17 @@ def test_xlsx_template_creation():
     for rs in rss:
         tk205.template(rs,'build/templates')
 
+def test_json_yaml_translation():
+    example_dir = 'build/examples/json'
```

Comment:
Do you want to reference "build/examples" or "schema-205/examples"?

Reply:
I think both are valid, but I have changed it to "schema-205/examples" since this will make the test file more robust. Using "build/examples" will only work if this test takes place after the 'test_json_roundtrip()" test takes place.